onboarding/views.py

```diff
@@ -699,24 +701,28 @@ def list_by_section_employee(self, request, pk):
     def finished(self, request, pk):
         """
         :param request:
-        :param pk: this is section ID if method is GET or page ID of section it belongs to if the method is PATCH
+        :param pk: this is section ID if method is GET or page ID of section it
+        :belongs to if the method is PATCH
```

Comment:
Our styles :smile: :wink: 
My `IDE` suggests there should not be `:` because it is for parameters ;-)
It says/hints:  `        belongs to if the method is PATCH`

Reply:
OK, I changed it only because comment exceeded 80 chars, didn't know what ":" meant